lua/mason-nvim-dap/mappings/adapters/haskell.lua

```diff
@@ -0,0 +1,5 @@
+return {
+  type = 'executable',
+  command = vim.fn.exepath('haskell-debug-adapter'),
+  args = { '--hackage-version=0.0.39.0' },
```

Comment:
Is this needed?

Reply:
The args?

I don't think so but it was what was used in the nvim-dap examples. 

Since we know we are installing the latest verified version with mason, I guess it's not worth adding it.